Release 11.3.18

camscale opened the pull request.

camscale committed:
Release 11.3.18

camscale commented:
I've updated this to include https://github.com/gravitational/teleport/pull/29582 after merging it.

tcsc commented on CHANGELOG.md:
```diff
@@ -1,5 +1,20 @@
 
+## 11.3.18 (07/26/23)
+
+* Fixed enhanced recording of missing `session.command` events when PAM enabled. [#29030](https://github.com/gravitational/teleport/issues/29030) [#29582](https://github.com/gravitational/teleport/pull/29582)
```
Optional: I'd consider trimming the changelog down a bit to user-focussed changes. Things like changing the Go version are possibly a bit developer centric, but it's not enough for me to block the merge.

camscale replied:
I included that because it was included in previous releases. I suspect so that people who track vulnerabilities can know we're building with a version without a vulnerability?

zmb3 commented:
Might want to ensure `e` ref is up to date before cutting the release, too.

camscale commented:
I've rebased the release/11.3.18 branch to include https://github.com/gravitational/teleport/pull/29575 which was merged after I branched. It is a performance improvement which I don't think warrants an entry in the changelog, so I'll just silently bring in the change.

I've also updated the `e` ref in this PR as requested by @zmb3.
